Marketplace/emoji-tablet/JSON_files/pickle.json

```diff
@@ -28,6 +28,6 @@
     "lifetime":30,
     "grabbable":true,
     "collisionless":false,
-    "script":"https://hifi-content.s3.amazonaws.com/elisalj/emoji_scripts/behaviors/crunch.js",
+    "script":"file:///D:/hifi-content/Marketplace/emoji-tablet/emoji_scripts/behaviors/crunch.js",
```

Comment:
Can you use `Script.resolvePath("../emoji_scripts/behaviors/crunch.js")` ?

Reply:
No, this is a JSON file and the comment explicitly calls out that these aren't final

Reply:
oops, I didn't see the comment at the top of this PR, sorry about that